package tps.proto;

message RT_TPS_Proto
{
    optional string uid = 1;
	required string parent_uid = 3;
}